src/tests/ftest/erasurecode/offline_rebuild_aggregation.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+'''
+import time
+from ec_utils import ErasureCodeIor, check_aggregation_status
+from apricot import skipForTicket
+
+class EcodAggregationOffRebuild(ErasureCodeIor):
+    # pylint: disable=too-many-ancestors
+    """
+    Test Class Description: To validate Erasure code object for offline rebuild
+                        with different Aggregation mode for Partial strip data.
+    :avocado: recursive
+    """
+    def test_ec_offline_rebuild_agg_disabled(self):
+        """Jira ID: DAOS-7313.
```

Comment:
General comment: Why we are having 3 different test cases with several repeated lines? A better solution would be have a common test_ec_offline_rebuild method and pass arguments from three different test cases.
example:
def test_ec_offline_rebuild(self, agg_flag="disabled"):
   .... All the steps on the method

Test Case 1:
def test_ec_offline_rebuild_agg_disabled(self):
     #Invoke test_ec_offline_rebuild
     test_ec_offline_rebuild(agg="disabled")

def test_ec_offline_rebuild_agg_disabled(self):
     #Invoke test_ec_offline_rebuild
     test_ec_offline_rebuild(agg="disabled")

def test_ec_offline_rebuild_agg_default(self):
     #Invoke test_ec_offline_rebuild
     test_ec_offline_rebuild(agg="enabled")

... so on...

In this way, you have a common code in one place and future modifications would be easy and can be done in one place.

Reply:
Yes, I can update the code but I have to go again with full Ci cycle for 2-3 days turn around :(